order/buyer/issue.md

```diff
@@ -0,0 +1,133 @@
+
+When the `/order-integration/order` API method returns HTTP status code 200, the order was successfully queued to be processed by Tradecloud. Processing takes usually less then a second, after which to order is available in the portal and is forwarded to the supplier ERP integration.
+
+{% hint style="info" %}
+After processing the order lines will have order process status `Issued`
+{% endhint %}
+
+### Send order API method
+
+{% api-method method="post" host="https://api.accp.tradecloud1.com/v2" path="/order-integration/order"%}
+{% api-method-summary %}Send order by buyer{% endapi-method-summary %}
+{% api-method-spec %}
+{% api-method-request %}
+{% api-method-headers %}
+{% api-method-parameter name="Authorization" type="string" required=true %} Bearer Access-Token {% endapi-method-parameter %}
+{% api-method-parameter name="Content-Type" type="string" required=true %} application/json {% endapi-method-parameter %}
+{% endapi-method-headers %}
+{% api-method-body-parameters %}
+{% api-method-parameter name="body" type="object" required=true %} Order JSON body {% endapi-method-parameter %}
+{% endapi-method-body-parameters %}
+{% endapi-method-request %}
+{% api-method-response %}
+{% api-method-response-example httpCode=200 %}
+{% api-method-response-example-description %} Successfully queued order (no body will be returned) {% endapi-method-response-example-description %}
+{% endapi-method-response %}
+{% endapi-method-spec %}
+{% endapi-method %}
+
+ [Send order OpenAPI Specification](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/order-integration/specs.yaml#/order-integration/sendOrderByBuyerRoute)
+
+### Order body JSON objects
+
+#### Order
+
+- `companyId`: your Tradecloud company identifier. You can find your company id in the URL when selecting "My company" in the portal dropdown menu. For example in `https://portal.accp.tradecloud1.com/company/06893bba-e131-4268-87c9-7fae64e16ee9` the last part `06893bba-e131-4268-87c9-7fae64e16ee9` is the company id.
+- `supplierAccountNumber`: the supplier account number as known in your ERP system
+- `purchaseOrderNumber`: the purchase order number as known in your ERP system
+- `destination`: the delivery destination of this order as known in your ERP system
+- `contact`: the employee responsible for this order. You can either send his/her email or userName as known in your ERP system
+  
+{% hint style="danger" %}
+`supplierAccountNumber`, `purchaseOrderNumber`, `destination.code`, `contact.email` and `contact.userName` should be unique within your company and never change. Never renumber or re-use numbers or code's.
+{% endhint %}
+
+{% hint style="danger" %}
+The `supplierAccountNumber` should be set on forehand in the Tradecloud connection with your supplier. You can set the account code when inviting a new connection or at any time in the connection overview in the portal.
+{% endhint %}
+
+#### Secondary order fields
```

Comment:
What are second order fields? Optional fields?

Reply:
Less import and optional